pkg/driver/nydus/builder/builder.go

```diff
@@ -79,7 +81,14 @@ func (builder *Builder) Run(option Option) (*Output, error) {
 		"--source-type",
 		"diff",
 		"--diff-overlay-hint",
+		"--fs-version",
+		option.RafsVersion,
+	}
+	if option.RafsVersion != "" {
+		// FIXME: these options should be handled automatically in builder (nydus-image).
+		args = append(args, "--disable-check", "--aligned-chunk")
```

Comment:
`aligned-chunk` is implied when `RafsVersion == 6`,

https://github.com/dragonflyoss/image-service/blob/e3381c6c0a595a644136bca0c389ee20731c6831/src/bin/nydus-image/main.rs#L463

Reply:
Fixed